src/Compilers/CSharp/Portable/Syntax/SyntaxFactory.cs

```diff
@@ -1545,12 +1545,14 @@ public static IdentifierNameSyntax IdentifierName(string name)
         /// </summary>
         public static SyntaxTree SyntaxTree(SyntaxNode root, ParseOptions? options = null, string path = "", Encoding? encoding = null)
         {
+#pragma warning disable RS0030 // Do not used banned APIs
             return CSharpSyntaxTree.Create((CSharpSyntaxNode)root, (CSharpParseOptions?)options, path, encoding);
```

Comment:
📝 This is part of a public API. Should it be updated to call the overload of `Create` that takes a specific checksum algorithm (e.g. Sha1)?

Reply:
Done.